8/README.md

```diff
@@ -0,0 +1,139 @@
+---
+domain: rfc.tango-controls.org
+shortname: 8/SERVER
+name: Device Server Model
+status: raw
+editor: 
+contributors: Piotr Goryl, Vincent Hardion
+---
+
+This document provides the specification of the Device Server.
+
+See also: 2/Device
+
+
+## Preamble
+
+Copyright (c) 2019 Tango Community.
+
+This Specification is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation; either version 3 of the License, or (at your option) any later version. This Specification is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with this program; if not, see <http://www.gnu.org/licenses>.
+
+This Specification is a [free and open standard](http://www.digistan.org/open-standard:definition) and is governed by the Digital Standards Organization's [Consensus-Oriented Specification System](http://www.digistan.org/spec:1/COSS).
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119).
+
+
+## Tango Device Server specification
+
+ The specication of '''Device Server''' aims the process of Device management, being it self a Device.
+
+### Goals
+
+ A Device Server in a Tango Controls System aims to control the process of communication and lifecycle of one or several Device.
```

Comment:
Suggest to rephrase as "In a Tango Control System a Device Server is in charge of managing the life cycle and the communication process of one or several Tango Devices.

Reply:
Sounds good !